x/mint/proto/mint/v1beta1/types.proto

```diff
@@ -22,67 +22,57 @@ message Params {
   // or 4 for weekly
   uint64 emission_calibrations_timestep_per_month = 3;
 
-  // percentage of the total supply that is rewarded to
-  // cosmos network validators the rest goes to
-  // allora reputers workers etc
-  string validators_vs_allora_percent_reward = 4 [
-    (cosmos_proto.scalar) = "cosmos.LegacyDec",
-    (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
-    (gogoproto.nullable) = false,
-    (amino.dont_omitempty) = true
-  ];
-
   // maximum total supply of the coin
-  string max_supply = 5 [
+  string max_supply = 4 [
```

Comment:
we should be careful about re-indexing, this would prob cause a state halt, requiring us to do something like

"allorad version 0.0.1 until block 54. then switch clients and continue to sync until"

Reply:
Good to know 

I think this is a major change and will require a new genesis for the major-upgrade PR but after that I will stop reindexing

Reply:
we def were gonna have to do that anyway so you're clear to do that up until monday